Sources/MapboxNavigation/InstructionLabel.swift

```diff
@@ -25,13 +25,19 @@ open class InstructionLabel: StylableLabel, InstructionPresenterDataSource {
     var customTraitCollection: UITraitCollection?
 
     var spriteRepository: SpriteRepository = .shared
-    
+
     var instruction: VisualInstruction? {
         didSet {
             updateLabelAttributedText()
         }
     }
-
+    
+    @objc public override var showHighlightedTextColor: Bool {
```

Comment:
We should remove this override `showHighlightedTextColor`. Because the original property defined in `StylableLabel` would trigger this work as well.